samples/apps/autogen-studio/frontend/src/components/atoms.tsx

```diff
@@ -1080,6 +1077,12 @@ export const AgentFlowSpecView = ({
   const [localFlowSpec, setLocalFlowSpec] =
     React.useState<IAgentFlowSpec>(flowSpec);
 
+  // Required to monitor localAgent updates that occur in GroupChatFlowSpecView and reflect updates.
```

Comment:
Excellent catch @junkei-okinawa!
Thanks! Will merge this into the dev branch and request your review for merging into main.